corehq/apps/registration/templates/registration/email/mobile_worker_confirm_account.html

```diff
@@ -0,0 +1,38 @@
+{% extends "registration/email/confirm_account.html" %}
+{% load i18n %}
+
+{% block preview_text %}
+  {% blocktrans %}
+    You have been added to the {{ domain }} project on CommCare HQ.
```

Comment:
Extract "CommCare HQ", like in [web_users.html](https://github.com/dimagi/commcare-hq/blob/985100b7b6b479b7180d5d2543d351c948680330/corehq/apps/users/templates/users/web_users.html#L14)

Reply:
https://github.com/dimagi/commcare-hq/pull/26946/commits/6d83722c0ae553d13a322ff906d045acd8a99720